### ISSUE TITLE
downstream task semseg

### ISSUE THREAD

**Issue by JunhyeopLee** [facebookresearch/ContrastiveSceneContexts#8]
Hello,

I have a question on semseg downstream task on the Stanford dataset.

Thanks to provide all the log files and pretrained models.
But Although dir or norm losses seem to be used for the semseg downstream task on the Stanford dataset as shown in your log file, there's no part to produce dir or norm losses in the 'downstream/semseg/lib/ddp_trainer.py, line 270-272'.
To reproduce your work on the Stanford dataset, should we modify dataset.py and ddp_trainer.py to include those loss terms?
(As I checked the ScanNet semseg log file, I found that those dir and norm loss terms are not used, unlike Stanford semseg task)

Thanks in advance.

**Comment by JunhyeopLee** [comment]
Thank you for answering.

There are a few more questions.

1. Then, CSC additionally used dir and norm losses and partitioning technique compared to the PointContrast (ECCV20) for Stanford semseg task?

2. If it is true, where can I find those dir and norm loss information? Could you recommend related papers? I am sorry that I couldn't find any related info in your paper..

3. Lastly, in table 10 in the supplementary material of your paper, all three models (scratch, PointContrast, and CSC) are finetuned by using sem loss as well as dir and norm losses?

Thanks!

**Comment by Sekunde** [comment]
Hello,

1. Yes.
2. We actually took it from PointGroup code/paper for instance segmentation purposes.
3. For scratch, and PointContrast in Table.10, we directly took numbers from the PointContrast paper, as we didn't observe an improvement w/ dir and norm loss when training from scratch and PointContrast.

**Comment by JunhyeopLee** [comment]
Thank you for answering!

**Comment by Sekunde** [comment]
@JunhyeopLee 
Hi, I use the current codebase (downstream/semseg, w/o norm and dir loss) to re-train a stanford semantic segmentationt task with [our model](http://kaldir.vc.in.tum.de/3dsis/contrastive_scene_contexts/pretrain/partition8_4096_100k.pth) as network initialization.
I got 72.5 miou and as reference I put the [curves](https://tensorboard.dev/experiment/JZip5XrGQ6u6KU5Uau0Duw/#scalars&_smoothingWeight=0), and [log file](http://kaldir.vc.in.tum.de/3dsis/contrastive_scene_contexts/finetune/stanford_semseg_partition8/log.txt) as well as [pre-trained mode](http://kaldir.vc.in.tum.de/3dsis/contrastive_scene_contexts/finetune/stanford_semseg_partition8/checkpoint_Res16UNet34C_miou.pth) here.

**Comment by JunhyeopLee** [comment]
@Sekunde 
Thank you for the updates! I will check it out! 
Thanks again! :D
